scripts/Makefile

```diff
@@ -90,8 +80,42 @@ ${GEOCINT_WORK_DIRECTORY}/geocint/installation/database_block_finished: ${GEOCIN
 	echo "Database initialisation step finished successfully"
 	touch $@
 
+# ------------------------ WEB Dashboard installation block ----------------------------
+${GEOCINT_WORK_DIRECTORY}/geocint/installation/nginx_installation: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/preparation_block_finished  ## install and configure nginx
+	sudo systemctl stop apache2
+	sudo systemctl disable apache2
+	sudo apt update
+	sudo apt install nginx
+	sudo ufw allow "Nginx Full"
+	sudo nginx -t
+	sudo systemctl start nginx
+	sudo systemctl restart nginx
+	touch $@
+
+${GEOCINT_WORK_DIRECTORY}/geocint/installation/set_modes: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/preparation_block_finished  ## create public_html folder and set access
+	sudo usermod -aG adm ${USER}
+	sudo usermod -aG ${USER} www-data
+	sudo mkdir -p ${GEOCINT_WORK_DIRECTORY}/public_html
+	sudo chown ${USER}:${USER} ${GEOCINT_WORK_DIRECTORY}/public_html
+	sudo chmod 0750 ${GEOCINT_WORK_DIRECTORY}/public_html
+	sudo echo 'export PATH=$$PATH:/usr/local/pgsql/bin' >> ${HOME}/.bashrc
```

Comment:
why is this necessary on packaged postgres?

Reply:
you are right, it's redundant

Reply:
@Zkir can you remove this line?

Reply:
Removed.